dusty/commands/setup.py

```diff
@@ -109,6 +117,8 @@ def setup_dusty_config(mac_username=None, specs_repo=None, nginx_includes_dir=No
     else:
         boot2docker_vm_memory = _get_boot2docker_vm_size()
 
+    _check_ssh_add()
```

Comment:
If we're telling the user to do things, it should be the last thing we output from the setup command, should be very visible (####HEY LOOK AT ME####), and should have explicit instructions on what to do.